nucleus/dataset.py

```diff
@@ -189,21 +251,19 @@ def autotag_training_items(self, autotag_name):
         )
         return response
 
-    def info(self) -> dict:
-        """Retrieve information about the dataset.
+    def info(self) -> DatasetInfo:
```

Comment:
[Here in \_\_init__.py](https://github.com/scaleapi/nucleus-python-client/blob/20060b692327775ae34976cc4fd347d946367207/nucleus/__init__.py#L3), will need to import and include this in the package `__all__`

Reply:
Will add. We should also look into using something like https://sphinx-pydantic.readthedocs.io/en/latest/ for simpler docstring generation from the `pydantic` models.